packages/preview-middleware-client/src/flp/init.ts

```diff
@@ -317,6 +324,9 @@ export async function init({
         await registerComponentDependencyPaths(JSON.parse(appUrls), urlParams);
     }
 
+    // Load rta connector
```

Comment:
@schreckstefan why did you add this here?

Reply:
@testojs please check if has any negative effects on ADP

Reply:
> @testojs please check if has any negative effects on ADP

Should not be an issue.